src/Infrastructure.Styles.Analyzer/RMSYN1008NoWhitespaceInAttributeAnalyzer.cs

```diff
@@ -0,0 +1,72 @@
+// This file is part of the re-motion Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// re-motion Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Infrastructure.Styles.Analyzer
+{
+  [DiagnosticAnalyzer(LanguageNames.CSharp)]
+  public class RMSYN1008NoWhitespaceInAttributeAnalyzer : DiagnosticAnalyzer
+  {
+    public const string DiagnosticId = "RMSYN1008";
+
+    internal const string Title = "Invalid number of whitespaces in attribute";
+    internal const string Message = "There must be no whitespace in an attribute";
+    internal const string Description = "Attribute must no contain a space";
```

Comment:
The wording should better describe the error. This analyzer checks whitespaces between the identifier and the optional argumentlist.